includes/admin/class-amp-editor-blocks.php

```diff
@@ -11,6 +11,13 @@
  */
 class AMP_Editor_Blocks {
 
+	/**
+	 * The ideal image size for the Latest Stories block.
+	 *
+	 * @var string
+	 */
+	public $image_size = array( 300, 400 );
```

Comment:
For most of the stories [here](https://www.ampproject.org/stories) that have a `poster-portrait-src` image, that image's dimensions are 600 x 800 px, like [this story](https://mic.com/stories/262/the-art-of-an-organized-refrigerator). So this is half of that.

Of course, this isn't a common dimension in WordPress.

Reply:
What about re-using the dimensions defined here:

https://github.com/ampproject/amp-wp/blob/3f570168fdfc9b4f0713089ad401852f52e88469/includes/class-amp-story-post-type.php#L86-L96

Reply:
Thanks, great idea. f325810d80c6b8dceb60ab913563e78fa77df121 uses that size `'amp-story-poster-portrait'`.